### DISCUSSION TITLE
Spice Guest Tools for a macOS Monterey Guest

### DISCUSSION THREAD

**Discussion by gschoen57** [utmapp/UTM#3772] [UTM Help (macOS)]
Hi everybody,

I just installed a macOS Monterey Guest on my M1 Mac, however I cannot use shared folders due to
missing spice guest tools. Are thery any for a macOS guest? I could not find anything on the internet.
Perhaps you have a link for me or can tell me how to install it.

Another thing is: I have connection to the internet in my virtual machine, but cannot 
login with my apple id - it always says "chack filed" and "the action could not be completed"

**Comment by socialsorcery** [comment]
Sadly, I believe the Apple ID issue is due to the virtualisation framework not faking the right device identifiers, see [this issue](https://github.com/utmapp/UTM/issues/3617). Not sure about the guest tools either.

**Reply by git0uttaHereIci** [reply]
Problem: I have a similar issue with guest tools/ file sharing / shared clipboard / copy / paste 

Env:
Host System: MacOS Monterey 12.4 / ARM64 M1  
Guest System: Debian 11.3 / ARM64

 Problem Again:  need shared file folder functionality between guest and host, 

Error msg: must install spice tools

Problem again: documentation is done by a bunch crackheads and alcoholics.. 

is there a pointer anywhere?

**Reply by git0uttaHereIci** [reply]
https://mac.getutm.app/support/

**Reply by git0uttaHereIci** [reply]
it's end user error, I don't have X11 installed

**Reply by kaystrobach** [reply]
same here - how to install the vm additions for osx? no download for osx on this site. ...

**Reply by nouranabualam** [reply]
Did any of you were able to download spice guest tools on your mac?

**Reply by Phobia-Cosmos** [reply]
so have you found one to solve that?

**Reply by mgoodlo3** [reply]
This is the solve that worked for me. https://www.youtube.com/watch?v=jPsrI5fS8zc
